src/ui/src/app/chart-details/chart-details-info/chart-details-info.component.ts

```diff
@@ -50,10 +50,17 @@ export class ChartDetailsInfoComponent implements OnInit {
   }
 
   maintainerUrl(maintainer: Maintainer): string {
-    if (this.chart.attributes.repo.source.match(/github.com/)) {
+    if (this.isKubernetesCharts(this.chart.attributes.repo.URL)) {
```

Comment:
So let me understand the assumptions made here.

Before, a source was set in the repo info. Now we assume that the only two repositories that we know for sure that have a github repo are the upstream one no?

If that's the case, I'll add a comment explaining the assumption

Reply:
Yes, that's the assumption we're making here. Will add a comment!